crates/uv/src/commands/build.rs

```diff
@@ -225,6 +230,27 @@ async fn build_impl(
         store_credentials_from_url(url);
     }
 
+    // Read build constraints.
+    let build_constraints =
+        operations::read_constraints(build_constraints, &client_builder).await?;
+
+    // Collect the set of required hashes.
+    // Enforce (but never require) the build constraints, if `--require-hashes` or `--verify-hashes`
+    // is provided. _Requiring_ hashes would be too strict, and would break with pip.
+    let build_hasher = HashStrategy::from_requirements(
+        std::iter::empty(),
+        build_constraints
+            .iter()
+            .map(|entry| (&entry.requirement, entry.hashes.as_slice())),
+        Some(&interpreter.resolver_markers()),
+        HashCheckingMode::Verify,
```

Comment:
So... this will validate any hashes that are provided in the constraints file. But it doesn't _require_ hashes. Should we add a separate flag to require them?

Reply:
I'd love that.

❤️ ❤️ ❤️